tests/org.jboss.tools.jst.js.node.test/src/org/jboss/tools/jst/js/node/test/NodeUITestCase.java

```diff
@@ -40,4 +45,51 @@ public static SWTBotView getProjectExplorer() {
 		return view;
 	}
 	
+	// JBIDE-20989 Preference validation fails on windows if node executable called node64.exe
+	public void testNode64ExePreferenceValidation() {
+		getProjectExplorer().show();
+		bot.menu("Window").menu("Preferences").click(); //$NON-NLS-1$ //$NON-NLS-2$
+		bot.waitUntil(Conditions.shellIsActive("Preferences"), NodeAllTests.DEFAULT_TIMEOUT); //$NON-NLS-1$
+		bot.tree().expandNode("JavaScript Tools").select("Node"); //$NON-NLS-1$ //$NON-NLS-2$
+		
+		IFolder nodeInstallationFolder = testProject.getFolder(getNodeInstallationFolder());
+		assertNotNull("Can't load JBIDE-20989 folder", nodeInstallationFolder); //$NON-NLS-1$
```

Comment:
getFolder() never returns null. So you can remove this assertNotNull. It's enough to check if nodeInstallationFolder.exists()